ejpcsvparser/utils.py

```diff
@@ -95,6 +95,13 @@ def convert_to_xml_string(string):
     return string
 
 
+def escape_angle_brackets(string):
```

Comment:
Would currently raise an exception if a 'truthy' non-string value is passed in, catching the `AttributeError` instead of the `if` statement would handle this:

```
def escape_angle_brackets(string):
    try:
        return string.replace('<', '&lt;').replace('>', '&gt;')
    except AttributeError:
        return string
```

Reply:
I considered an exception rather than only accounting for ``None`` values, I will change it over to the exception as you suggest - thanks @seanwiseman !